.github/workflows/terraform.yml

```diff
@@ -53,7 +53,15 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           TF_VAR_github_sha: ${{ github.sha }}
         run: terraform plan
-
+  terraform-security-scan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Terraform security scan
+        uses: triat/terraform-security-scan@v1
```

Comment:
Can we lock this to a sha instead of a version? 

https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsuses

While locking to a major version will

>Using the specific major action version allows you to receive critical fixes and security patches while still maintaining compatibility. It also assures that your workflow should still work.

There is still the possibility that this action can be hijacked and malicious code can be run. 

@obrien-j thoughts?

Reply:
Alternatively we can fork and own this repo.